### DISCUSSION TITLE
Episode 119: Why APL & Haskell are AWESOME with Zach Laine! (Part 3)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#10] [Episode]
Episode Link: https://adspthepodcast.com/2023/03/03/Episode-119html

Feel free to write comments or ask questions 🙂

**Comment by standard-librarian** [comment]
Anyone knows what's that country in the episode.

**Reply by codereport** [reply]
What do you mean which "country"? Do you have a timestamp?
